ui/decredmaterial/collapsible.go

```diff
@@ -9,277 +9,159 @@ import (
 	"gioui.org/widget/material"
 )
 
-type MoreItem struct {
-	Text   string
-	Button *widget.Clickable
-	label  Label
-}
-
-type Collapsible struct {
-	IsExpanded      bool
-	Button          *widget.Clickable
-	BackgroundColor color.RGBA
-	expandedIcon    *widget.Icon
-	collapsedIcon   *widget.Icon
+type collapsibleCommon struct {
+	isExpanded      bool
+	button          *widget.Clickable
+	BackgroundColor color.NRGBA
```

Comment:
I don't think this is necessary. kindly look at how different button types were implemented in gio widgets here https://git.sr.ht/~eliasnaur/gio/tree/main/item/widget/material/button.go. Separate the structs for the two collapsible types and reuse code where you can.